src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
- [ ] one of your test is failing, kindly fix this
![image](https://github.com/fickryiman/Free-PC-Games/assets/66097371/1d244226-0e21-4835-a294-98f57a248370)

Reply:
fixes integration test, removed test files from create-react-app boilerplate

Reply:
![integration test free-pc-games](https://github.com/fickryiman/Free-PC-Games/assets/116701353/5d7f898b-b665-4a63-9c96-a2c3d2103ff4)